### PR TITLE
Made some UI refinements

### PR DESCRIPTION
**Pull request by soumyamahunt**
Added separator between hamburger button and tabs. Compact overlay now opens with maximum height and width.

## PR Type
What kind of change does this PR introduce?

Feature

## Other information
cmxiTInXru
KKP2S8BxrL

### CODE REVIEW

**Comment by JasonStein** [comment]
I don't think that search apply button is necessary. It is kind of counter intuitive. Can you revert the changes for that part? Also remove the tooltip string since it won't be needed. Rest are looking good.

**Comment by soumyamahunt** [comment]
> I don't think that search apply button is necessary. It is kind of counter intuitive. Can you revert the changes for that part? Also remove the tooltip string since it won't be needed. Rest are looking good.

Reverted those changes.

File: src/Notepads.Controls/SetsView/SetsView.cs [hunk 1]
```diff
@@ -223,7 +223,17 @@ private void SetsView_SelectionChanged(object sender, SelectionChangedEventArgs
 
         private void UpdateSetSeparators()
         {
-            for (int i = 0; i < Items?.Count; i++)
+            int i = 0;
+            if (SelectedIndex != i)
```
Comment (JasonStein) [inline src/Notepads.Controls/SetsView/SetsView.cs:227]: use != 0 here. "!= i" might have some side effect in the future. Just move int i = 0 into the for loop and use "0" for the first if-else block.

Comment (soumyamahunt) [reply src/Notepads.Controls/SetsView/SetsView.cs]: Made changes.

**Pull request merged**